src/status_im/transport/db.cljs

```diff
@@ -0,0 +1,29 @@
+(ns status-im.transport.db
+  (:require-macros [status-im.utils.db :refer [allowed-keys]])
+  (:require [cljs.spec.alpha :as spec]))
+
+;; required
+(spec/def ::ack (spec/coll-of string? :kind vector?))
+(spec/def ::seen (spec/coll-of string? :kind vector?))
+(spec/def ::pending-ack (spec/coll-of string? :kind vector?))
```

Comment:
Can we document these in docstrings? It isn't immediately clear what ack/seen/pending-ack/send is.

Reply:
More generally in file as well, e.g. not clear what discovery-filter is exactly from this ns context

Reply:
Actually none of those is used for now, it was for when we add acks if we do at some point

Reply:
That's fine, even more reason to document :)